webapp/src/language/es.json

```diff
@@ -383,17 +384,17 @@
       "imDone": "Listo"
     },
     "credentialsRecovery": {
-      "credentialsRecovery": "Recuperación de credenciales",
+      "passwordRecovery": "Recuperación de Contraseña",
       "emailError": "El correo ingresado no esta asociado a ninguna cuenta",
       "changePasswordInstructions": "Para cambiar su contraseña ingrese el correo asociado a su cuenta, la contraseña la actual y la nueva",
       "changePassword": "Cambiar contraseña",
       "currentPassword": "Contraseña actual",
       "errorPassword": "La contraseña actual no coinside con su correo",
       "newPassword": "Nueva contraseña",
-      "instructionCredentialsRecovery": "Ingrese el correo asociado a su cuenta para poder recuperarla",
+      "instructionCredentialsRecovery": "Use su correo electrónico registrado en Lifebank para recuperar su contraseña. Le enviaremos un correo para que cree una nueva.",
```

Comment:
English: to set up a new password.
Spanish: cree una nueva.

I think is missign **_contraseña_** word